dv/uvm/data/vcs/vcs.hjson

```diff
@@ -31,13 +31,13 @@
                "-LDFLAGS \"-Wl,--no-as-needed\""]
 
   run_opts:   ["-licqueue",
-               "-ucli -do {tool_srcs_dir}/vcs_fsdb.tcl",
+               "-ucli -do {tool_srcs_dir}/vcs_{dump}.tcl",
```

Comment:
This (confusingly) needs to be `vcs_fsdb.tcl` at the moment.

Reply:
ah yes good point